src/Centreon/Domain/Repository/Traits/CheckListOfIdsTrait.php

```diff
@@ -42,14 +42,27 @@ trait CheckListOfIdsTrait
 {
 
     /**
-     * 
+     * Check a list of IDs
+     *
      * @param array $ids
-     * @param string $tableName
-     * @param string $columnNameOfIdentificator
+     * @param string $tableName not needed if entity had metadata
+     * @param string $columnNameOfIdentificator not needed if entity had metadata
      * @return bool
      */
-    protected function checkListOfIdsTrait(array $ids, string $tableName, string $columnNameOfIdentificator): bool
+    protected function checkListOfIdsTrait(
+        array $ids,
+        string $tableName = null,
+        string $columnNameOfIdentificator = null
+    ): bool
```

Comment:
```suggestion
    ): bool {
```
remove warning from phpcs